toolkit/docs/formats/imageconfig.md

```diff
@@ -85,6 +85,10 @@ SystemConfigs is an array of SystemConfig entries.
 
 SystemConfig defines how each system present on the image is supposed to be configured.
 
+### IsKickStartBoot
```

Comment:
Probably want to add a comment under `### PreInstallScripts` that this is also required to run the scripts.

Reply:
Fixed